kie-performance-kit/src/main/java/org/kie/perf/metrics/PerfRepoReporter.java

```diff
@@ -110,7 +110,8 @@ public void report(SortedMap<String, Gauge> gauges, SortedMap<String, Counter> c
             }
         }
 
-        String testUid = tc.getProjectName().toLowerCase().replaceAll(" ", "_") + "_" + tc.getSuite().toLowerCase() + "_" + tc.getScenario().toLowerCase();
+        String testUid = tc.getProjectName().toLowerCase().replaceAll(" ", "_") + "_"
+                + tc.getSuite().toLowerCase() + "_" + tc.getScenario().toLowerCase() + "_" + tc.getTestUIDSuffix();
```

Comment:
Can you modify this just a little bit so in case a suffix is not provided, the last `_` is not appended?

Reply:
I added an if statement there